web/elm/src/SideBar/Styles.elm

```diff
@@ -330,7 +340,7 @@ tooltip top left =
 
 tooltipBody : List (Html.Attribute msg)
 tooltipBody =
```

Comment:
Looks like the tooltip text has the same colour as the background:

<img width="220" alt="Screen Shot 2020-11-03 at 10 14 46 AM" src="https://user-images.githubusercontent.com/26583442/98004351-d553de80-1dbd-11eb-93c5-6d3e23858eac.png">

Reply:
GOOD SNAG! :D